src/calculation/HTMLBuilder.ts

```diff
@@ -54,11 +69,28 @@ Handlebars.registerHelper('highlightClause', (localId, context) => {
   return '';
 });
 
+// apply highlighting style to covered clauses
+Handlebars.registerHelper('highlightCoverage', (localId, context) => {
+  const libraryName: string = context.data.root.libraryName;
+  const clauseResults: ClauseResult[] = context.data.root.clauseResults;
+
+  const clauseResult = clauseResults.filter(result => result.libraryName === libraryName && result.localId === localId);
+  if (clauseResult) {
+    if (clauseResult.some(c => c.final === FinalResult.TRUE)) {
+      return objToCSS(cqlLogicClauseCoveredStyle);
+    } else if (clauseResult.every(c => c.final === FinalResult.FALSE)) {
```

Comment:
True, I forgot that we had reasoned that this was a safe assumption since the clauses have been filtered just to be relevant clauses. I can try with an else statement and see what the results look like!

Reply:
I'm getting the same results with `else if` and `else` so I do think we can go along with that assumption